doc/example.rst

```diff
@@ -23,6 +23,7 @@ Getting started
 .. toctree::
    :maxdepth: 2
 
+   example/getting_started.ipynb
```

Comment:
Cannot make a comment within getting_started, therefore here:

I think Software Development Standard and Further topics are within the notebook twice

Reply:
At least I only see it in there one ^^